library/Cube/CubeRenderer/HostStatusCubeRenderer.php

```diff
@@ -69,6 +69,10 @@ public function renderFacts($facts)
         $main = '';
         $sub = '';
         foreach ($parts as $class => $count) {
+            if ($count === 0) {
```

Comment:
Why?

Reply:
If the `Problems only` control is enabled, it may cause the `OK|UP` counter to be 0, we don't need to render it.

Reply:
It could be zero without the `Problems only` control as well, right?

Reply:
Yes, you are right. So we need it anyway.